### DISCUSSION TITLE
Feature: playout functionality for Webradio

### DISCUSSION THREAD

**Discussion by rclabs-dev** [jcorporation/myMPD#787] [Ideas]
Hi dear,

This application is awesome. 

However, it could cover a wider range of users if there is a playout system, which basically generates a playlist based on 24 hours of the day. So, as an example, the user will be able to program a certain program for 9 pm and with a duration of 1 hour, where he will be able to add songs individually or from playlists. In this way the user would have a calendar based on day/time to perform the daily schedule. At times when no program was scheduled, the program would continue playing music from a playlist chosen for this function.

It would be useful for webradio applications, once the MPD already sends the stream to Icecast/Shoutcast.

Best regards,

Regivaldo Costa

**Comment by jcorporation** [comment]
I do not fully understand your proposed playout functionality.

myMPD provides:
- timers
- scripting
- playlists
- smart playlists
- jukebox mode

This features should covering your playout functionality?

**Reply by rclabs-dev** [reply]
Hi dear,

Sorry by the delay, was traveling.

I did explore more the MyMPD. Timers is a function like playout I said and it works well. Sorry by this thread.

I lacked attention in my exploration. I will test the full functionality of the MyMPD.

Many thanks for your reply.

Best regards,

Regivaldo Costa
--
